core/platform-api/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/WorkspaceManagerTest.java

```diff
@@ -106,10 +109,13 @@ public User getUser() {
 
     @Test
     public void shouldBeAbleToCreateWorkspace() throws Exception {
+        // given
```

Comment:
Why did you do that?
As you can see _given, when, then_ sections are separated with an empty line which is enough to understand the test, i try to follow this pattern and avoid explicit comments.